integ-test/src/test/java/org/opensearch/sql/ppl/TrendlineCommandIT.java

```diff
@@ -75,4 +75,71 @@ public void testTrendlineWithSort() throws IOException {
                 TEST_INDEX_BANK));
     verifyDataRows(result, rows(new Object[] {null}), rows(44313.0), rows(39882.5));
   }
+
+  @Test
+  public void testTrendlineWma() throws IOException {
+    final JSONObject result =
+        executeQuery(
+            String.format(
+                "source=%s | where balance > 39000 | sort balance | trendline wma(2, balance) as"
+                    + " balance_trend | fields balance_trend",
+                TEST_INDEX_BANK));
+    System.out.println("Result (Base): " + result.toString());
+    verifyDataRows(
+        result, rows(new Object[] {null}), rows(45570.666666666664), rows(40101.666666666664));
+  }
+
+  @Test
+  public void testTrendlineMultipleFieldsWma() throws IOException {
+    final JSONObject result =
+        executeQuery(
+            String.format(
+                "source=%s | where balance > 39000 | sort balance | trendline wma(2, balance) as"
+                    + " balance_trend wma(2, account_number) as account_number_trend | fields"
+                    + " balance_trend, account_number_trend",
+                TEST_INDEX_BANK));
+    verifyDataRows(
+        result,
+        rows(null, null),
+        rows(40101.666666666664, 16.999999999999996),
+        rows(45570.666666666664, 29.666666666666664));
+  }
+
+  @Test
+  public void testTrendlineOverwritesExistingFieldWma() throws IOException {
+    final JSONObject result =
+        executeQuery(
+            String.format(
+                "source=%s | where balance > 39000 | sort balance | trendline wma(2, balance) as"
+                    + " age | fields age",
+                TEST_INDEX_BANK));
+    System.out.println("Result (Overwrite) : " + result.toString());
+    verifyDataRows(
+        result, rows(new Object[] {null}), rows(40101.666666666664), rows(45570.666666666664));
```

Comment:
Can you just do this? Looks like some other tests have this?

```suggestion
        result, rows(null), rows(40101.666666666664), rows(45570.666666666664));
```

Reply:
This method signature of `rows( )` is defaulted to read an array, and I reckon we should keep that by supplying an array for the method invoke, or explicit to specify multiple `null` like what test-case `testTrendlineMultipleFields` do.

Because `...` is being used, so specifying a single `null` will mis-lead the compiler which `null` will be assigned to `expectedObjects`, in contrast to the case of  `null, null` and `new Object[] {null}` which both hint compiler an valid Object array is being passed in, with content `null`.

 
<img width="745" alt="image" src="https://github.com/user-attachments/assets/68a3e8f2-8986-4d4c-b123-c85911697404" />